Code_of_Conduct.md

```diff
@@ -0,0 +1,42 @@
+## Code of Conduct ##
+
+This project is dedicated to providing a welcoming and supportive environment for all people, regardless of background or identity. Members do not tolerate harassment for any reason, but especially harassment based on gender, sexual orientation, disability, physical appearance, body size, race, nationality, sex, color, ethnic or social origin, pregnancy, citizenship, familial status, veteran status, genetic information, religion or belief, political or any other opinion, membership of a national minority, property, age, or preference of text editor. 
```

Comment:
As long as we all know [which text editor is the true path to nirvana](https://www.gnu.org/fun/jokes/ed-msg.en.html).

Reply:
ngl, I do this often 

```
$ cat > my_file.txt
some text
good luck! no going back :)
^d
```